reconcile/terraform_vpc_resources/integration.py

```diff
@@ -164,7 +168,10 @@ def run(self, dry_run: bool) -> None:
             thread_pool_size=thread_pool_size,
         )
 
-        tf_client.plan(enable_deletion=enable_deletion)
+        disable_deletions, errors = tf_client.plan(enable_deletion=enable_deletion)
+
+        if disable_deletions or errors:
+            raise PlanStepError("Errors in terraform plan step, please verify output.")
```

Comment:
similar error handling in terraform resources https://github.com/app-sre/qontract-reconcile/blob/836f3bbb0a07648c2049d198a52dc4afaea0ada3/reconcile/terraform_resources.py#L479-L483 and several other places, maybe worth extracting and use the same flow

Reply:
I've moved inside `terraform_client`.